deu/overpayment-claim-entered.vbs

```diff
@@ -189,7 +186,6 @@ Do
 		IF HC_claim_amount = "" THEN err_msg = err_msg & vbNewLine &  "* Please enter the amount of claim."
 	END IF
     	IF EVF_used = "" then err_msg = err_msg & vbNewLine & "* Please enter verification used for the income received. If no verification was received enter N/A."
-	'IF isdate(income_rcvd_date) = False or income_rcvd_date = "" then err_msg = err_msg & vbNewLine & "* Please enter a valid date for the income received."
```

Comment:
Line 196 - will need to add a transmit to get to correct MEMB

Reply:
Line 465 - Remove msgbox. This interrupts the user experience. Find a way to handle for this in the closing message box or another way.

Reply:
Line 196 updated
Line 465 removed